src/ShallowTraversal.js

```diff
@@ -108,34 +107,34 @@ export function nodeMatchesObjectProps(node, props) {
   return isSubset(propsOfNode(node), props);
 }
 
+function buildSelectorPredicate(selector) {
+  const nodes = getAst(selector).nodes[0];
+  return AND(nodes.map(node => {
+    switch (node.type) {
+      case 'class':
```

Comment:
It's a bit nit, but the old way of using object constants seemed safer/nicer than just strings. Can you update this to use constants?